### ISSUE TITLE
strange 'thing' in bg

### ISSUE THREAD

**Issue by Collector-Hellscream** [HellscreamWoW/Tracker#4127]
It's pretty rare. Got that only 2-3 times. Sometimes when you die in battleground you respawn immedietly at the place of your dead with 1% health but you are not attackable, can't release, move or speak.

![wowscrnshot_010516_222416](https://cloud.githubusercontent.com/assets/14946340/12128164/e53114de-b3fb-11e5-92cc-f92a6f6420cf.jpg)
![wowscrnshot_010516_222429](https://cloud.githubusercontent.com/assets/14946340/12128163/e52f12c4-b3fb-11e5-819f-bf14273aedfc.jpg)

**Comment by EPS1L0N** [comment]
It happens when you die, not when you release right? and what happens if you relog?

**Comment by Collector-Hellscream** [comment]
Yes. I'm death for less than a second (so I can't be sure but I thing there is release button for a very short period of time) and respawn immeditly after that. Didn't relogged so can't tell what happens.
